demo/src/main/res/values/strings.xml

```diff
@@ -282,12 +278,12 @@
     <string name="component_tabs_add_tab">Add tab</string>
     <string name="component_tab_icon_leading">Leading</string>
     <string name="component_tab_icon_top">Top</string>
-    <string name="component_tab_favourites">Favourites</string>
-    <string name="component_tab_calls">Calls</string>
-    <string name="component_tab_alerts">Alerts</string>
-    <string name="component_tab_calendar">Calendar</string>
-    <string name="component_tab_account">Account</string>
-    <string name="component_tab_settings">Settings</string>
+    <string name="component_tab_coffee">Coffee</string>
+    <string name="component_tab_cooking_pot">Cooking Pot</string>
+    <string name="component_tab_ice_cream">Ice Cream</string>
+    <string name="component_tab_restaurant">Restaurant</string>
+    <string name="component_tab_favorites">Favorites</string>
+    <string name="component_tab_information">Information</string>
```

Comment:
Should we mutualize these data used by tabs and bottom navigation components by creating a data class with a title and a drawable resource?

Reply:
I created a `NavigationItem` enum used by both tabs and bottom navigation. I could not find a better name, another solution would be to rename it to `BottomNavigationItem` and add a typealias:

 ```kotlin
typealias TabItem = BottomNavigationItem
```

Reply:
I agree for type aliases. Let's keep `NavigationItem` for now.